### ISSUE TITLE
Make 'scrubbing' an official property

### ISSUE THREAD

**Issue by heff** [videojs/video.js#1766]
As a follow up to #1741, scrubbing should become an official player API property. Currently the progress bar sets `player.scrubbing = true`, and if we change that now it could break external control bars, so this should wait for 5.0.

Here's the code to make it happen. The progress bar will need to be updated.
```js
/**
 * Store the internal state of scrubbing
 * @private
 * @return {Boolean} True if the user is scrubbing
 */
vjs.Player.prototype.scrubbing_ = false;


/**
 * Returns whether or not the user is "scrubbing". Scrubbing is when the user 
 * has clicked the progress bar handle and is dragging it along the progress bar.
 * @param  {Boolean} isScrubbing   True/false the user is scrubbing
 * @return {Boolean}               The scrubbing status when getting
 * @return {Object}                The player when setting
 */
vjs.Player.prototype.scrubbing = function(isScrubbing){
  if (isScrubbing !== undefined) {
    this.scrubbing_ = !!isScrubbing;

    if (isScrubbing) {
      this.addClass('vjs-scrubbing');
    } else {
      this.removeClass('vjs-scrubbing');
    }

    return this; 
  }
  
  return this.scrubbing_;
};
```

**Comment by forbesjo** [comment]
Looking at this

**Comment by heff** [comment]
Closed by #2080
